assets/components/src/plugin-installer/index.js

```diff
@@ -177,15 +176,6 @@ class PluginInstaller extends Component {
 			? __( 'Activate' )
 			: __( 'Install' );
```

Comment:
This is true on `master` too, but it seems heightened to me with the new design. And maybe now's a good time to change it. 

1. (See the Disqus section for what I'm talking about.) If there's more than one plugin to be installed, should this primary action button say "Install All" instead of just "Install"? With just "install" it's not immediately clear to me how the action differs from clicking "Install" next to one of the individual plugins.

<img width="1078" alt="Screen Shot 2020-11-30 at 3 13 34 PM" src="https://user-images.githubusercontent.com/2230142/100672594-66679800-331f-11eb-95ed-9174e09dbaa4.png">

2. (See the Coral section for what I'm talking about.) If there is is only one plugin to be installed, do we even need the action button, since it will do the same thing as clicking "Install" next to the single plugin?

Reply:
I'm not even sure we need the buttons here.

Reply:
Removed in 1d4690bd37b54196720977c93e4805951c56b53e